zcutil/release-notes.py

```diff
@@ -0,0 +1,100 @@
+import re, sys, os, os.path
+import subprocess
+import argparse
+from itertools import islice
+from operator import itemgetter
+
+author_aliases = {
+    'Simon': 'Simon Liu',
+    'bitcartel': 'Simon Liu',
+    'EthanHeilman': 'Ethan Heilman',
+}
+
+def apply_author_aliases(name):
+    if name in author_aliases:
+        return author_aliases[name]
+    else:
+        return name
+
+def parse_authors(line):
+    commit_search = re.search('(\d+)', line)
+    if commit_search:
+        commits = commit_search.group(1)
+    else:
+        commits = 0
+    name = re.sub(' \(\d+\)|:|\n', '', line)
```

Comment:
Could this miss cases where the line ending is `\r\n`? Does this work using `$` instead to match the end of the line in regex?